release_notes_draft.md

```diff
@@ -18,4 +18,41 @@ now do a different and much cooler thing.
 
 # App
 
-* Adapt the height of select prompts to the height of the terminal so that prompts won't be cut off in shorter terminals.
+* *JS functions.* JavaScript functions are now available! You can learn more in
+the docs: https://shopify.dev/docs/apps/functions/language-support/javascript
+* *Theme app extension logs.* Logs for theme app extensions will no longer
+appear in a separate area and conflict with other logs or the preview/quit bar.
+Instead, they will be integrated with the other logs. Progress bars have been
+replaced with static output to accommodate this placement.
+* *Customer accounts UI extension preview.* The link from the dev console for
+Customer Accounts UI Extensions will now redirect to the customer account page.
+
+# Theme
+
+* *Theme environments.* Configure environments with theme and store in a
+`shopify.theme.toml` file in your project root. Use the `--environment` flag (or
+`-e` for short) to switch between environments. For more details, read the docs
+at https://shopify.dev/docs/themes/tools/cli
```

Comment:
We already know the final URL ([documentation PR](https://github.com/Shopify/shopify-dev/pull/32001))

```suggestion
at https://shopify.dev/docs/themes/tools/cli/environments
```